components/org.wso2.carbon.appmgt.gateway/pom.xml

```diff
@@ -78,6 +78,31 @@
             <artifactId>org.wso2.carbon.identity.sso.saml</artifactId>
          </dependency>
 
+        <dependency>
+           <groupId>org.wso2.carbon.mediation</groupId>
+           <artifactId>org.wso2.carbon.rest.api.stub</artifactId>
+       </dependency>
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/05d5cef1f9e3d525eb6d0c0af34943c96daac8ae